app/experimenter/experiments/models/nimbus.py

```diff
@@ -195,6 +195,24 @@ def __str__(self):
         return self.name
 
 
+class NimbusDocumentationLink(models.Model):
+    experiment = models.ForeignKey(
+        NimbusExperiment,
+        related_name="documentation_links",
+        on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=255, null=False)
+    link = models.URLField(max_length=255, null=False)
+
+    class Meta:
+        verbose_name = "Nimbus Documentation Link"
+        verbose_name_plural = "Nimbus Documentation Links"
+        ordering = ("id",)
+
+    def __str__(self):
+        return "%s (%s)" % (self.title, self.link)
+
+
```

Comment:
Oh I see, you did it as a 1:M, we did this in the old flow with a text field that people could paste arbitrary links into and then just `urlize` when we display it so they're all clickable.  Do we need to store them independently in the database?

Reply:
The links are independent title / link fields in the figma mocks

https://www.figma.com/file/5SLa07XqQb0ttB1ay1gami/?node-id=217143%3A15592

Reply:
> a text field that people could paste arbitrary links into

Not sure that would work since the mocks have us using separate title and link fields.

![Screen Shot 2021-01-11 at 12 48 48 PM](https://user-images.githubusercontent.com/6392049/104218866-5b6a5200-540b-11eb-8f4d-87c8bd259b40.png)

But that's just a UI thing; we could still store and parse a single JSON string if you want it in one DB column.

Reply:
Could rework this PR to just store a JSON blob of links on the backend, shift more of the data management to the frontend to still conform to the mocked UX. Or we could rethink the UX? This seemed like the most straightforward option though.

Reply:
Oh sorry yes I didn't look at the figma, I should've looked at that first 😅 Yeah I think if we don't need to do any rich logic on these individual links we could simplify this a lot with just a single text field of arbitrary text and highlight links in the UI.  Maybe we can ping @AnaMedinac and review the ui together?

Reply:
I'm partial to itemized title/link items personally. At least from a UX perspective it makes it clear what's expected. With a textbox you might not know if you can include regular text or what happens to it if you do use it. Additionally, with a text box you likely lose the ability to assign titles to links.

Reply:
Hmm I like document title/link as well FWIW, they'll be displayed in the Summary component with the title: 
<img width="332" alt="image" src="https://user-images.githubusercontent.com/13018240/104224422-94ee8d80-540a-11eb-915e-25cc7cd77e07.png">

But I'm sure Ana would be flexible.

Reply:
We discussed this in slack and captured the results [here](https://jira.mozilla.com/browse/EXP-802) tldr we'll use the 1:M approach and potentially change `title` to an enum drop down so we can richly validate/identify given links.